lib/src/components/NxTransferList/__tests__/NxTransferList.test.tsx

```diff
@@ -269,4 +271,101 @@ describe('NxTransferList', function() {
       expect(threeSelected.childAt(1)).toHaveProp('footerContent', 'To 3 and beyond!');
     });
   });
+
+  it('throw an error when passing a Set to selectedItems when allowReordering is true', function() {
+    expect(() => getShallow({ allowReordering: true }))
```

Comment:
This should no longer compile.  If it does, the props changes didn't work as expected

Reply:
> This should no longer compile. If it does, the props changes didn't work as expected

Sadly this did compile 😭

Reply:
> Sadly this did compile sob

Actually that might just be a problem with `getShallow`.  If I change `NxTransferListWithReorderingExample` to use a Set instead of an Array, it fails appropriately